op-challenger/cmd/watch/oracle.go

```diff
@@ -0,0 +1,76 @@
+package watch
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"os/signal"
+	"syscall"
+
+	log "github.com/ethereum/go-ethereum/log"
+
+	challenger "github.com/ethereum-optimism/optimism/op-challenger/challenger"
+	config "github.com/ethereum-optimism/optimism/op-challenger/config"
+	metrics "github.com/ethereum-optimism/optimism/op-challenger/metrics"
+)
+
+// Oracle listens to the L2OutputOracle for newly proposed outputs.
+func Oracle(logger log.Logger, cfg *config.Config) error {
```

Comment:
This would be cool if it optionally also took an op-node RPC url and then did a comparison, could be used as a detector service to trigger an alert if the mismatched output exists for too long

Reply:
Interesting, what do you think about putting this behind a long flag? (Maybe something like "--with-rpc")